### PR TITLE
Added missing translations in validators.tr.xlf

### PR DESCRIPTION
**Pull request by mertingen**
I referred the same tone and style in the new translations. I did not touch the abbreviation name such as IBAN or BIC. Because they have no counterparts.

| Q             | A
| ------------- | ---
| Branch?       | master for features / 3.4 up to 4.2 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| BC breaks?    | no     
| Deprecations? | yes/no 
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf [hunk 1]
```diff
@@ -222,6 +222,110 @@
...
+                <source>This is not a valid UUID.</source>
+                <target>Bu geçerli bir UUID değildir.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Bu değer {{compare_value}} değerinin katlarından biri olmalıdır.</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Bu İşletme Tanımlayıcı Kodu (BIC), IBAN {{iban}} ile ilişkili değildir.</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: Can you keep the whitespace after `{{` and before `}}`? (the same goes for other translations as well).

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf (hunk 1 above)
Comment (mertingen) [reply src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf]: I did. It's done.

**Review by javiereguiluz — APPROVED** [review]
👍 Verified with Google Translate.

Note to mergers: maybe we can sort the translations while merging? The file contained `1..57` and `82` translations and now it contains `1..85`, but `82` is at the end of the list. Thanks.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @mertingen.

**Pull request merged**